src/x_connective.c

```diff
@@ -615,6 +616,8 @@ static void route_list(t_route *x, t_symbol *sel, int argc, t_atom *argv)
         }
     }
  rejected:
+    if(argc == 0)
+        outlet_bang(x->x_rejectout);
```

Comment:
what is this doing?

Reply:
I guess this is a remainder of another PR. `outlet_list` below will already output the list resp. bang.

Reply:
ping @porres

Reply:
how about now?

Reply:
looks good now :-)